### ISSUE TITLE
Getting operationName and query string when an error occurs

### ISSUE THREAD

**Issue by CryptoGraffe** [graphql-editor/graphql-zeus#297]
Is there any way to get operationName and query string when there is an error? The GraphQLError object is pretty sparse.

**Comment by aexol** [comment]
Good one. We should output better errors

**Comment by CryptoGraffe** [comment]
Here are some of my thoughts on what this class should include:

1. The `operationName` and the `query` and `variables` and any other obvious context. 
2. more fields on the errors array, as `code` and `path` are usually there. 
3. The console.error() log is troublesome for things like automated error capture. 
4. The string should ideally elevate the underlying error message, rather than just `GraphQL Response Error`

**Comment by GauBen** [comment]
- 2 and 3 will be fixed by #321
- 4 is not that easy considering that there can be several errors, I tried to create a meaningful error message, but you might find room for improvement there

**Comment by akkie** [comment]
Yes, please do not log from inside the error. The calling code should handle the logging.
